### DISCUSSION TITLE
ModuleNotFoundError: No module named 'encodec'

### DISCUSSION THREAD

**Discussion by NitroNerd** [JonathanFly/bark#16] [Q&A]
Ho to fix this ?

> ModuleNotFoundError: No module named 'encodec'

Appears after this command
`python bark_perform.py --text_prompt "It is a mistake to think you can solve any major problems just with potatoes... or can you? (and the next page, and the next page...)" --split_by_words 35`

**Comment by viperwasp** [comment]
> Ho to fix this ?
> 
> > ModuleNotFoundError: No module named 'encodec'
> 
> Appears after this command `python bark_perform.py --text_prompt "It is a mistake to think you can solve any major problems just with potatoes... or can you? (and the next page, and the next page...)" --split_by_words 35`

I figured it out. I don't know if this is the "correct" way to solve it. 
Basically I looked up the error and it told me to do 

pip install encodec 

And then I ran the python bark_perform.py script the full one and still got a traceback. 
Next it asked for funcy I think 

So I did "pip install funcy"  
And then I ran the prompt and it wanted tqdm or transformers
Your getting picture right? I kept pip installing what it asked for next. 

4-5 pip installs later it was fine. I ran the command it did a lot of stuff in cmd. And then generated my outputs. It seems to be working. Good luck! I'm terrible at this stuff. Amazed I found a solution mostly on my own.

**Reply by JonathanFly** [reply]
I use the same technique. Oh missing X? Ok install X. Now missing Y? Ok install Y. This is a classic time-tested advanced software engineering technique. You are now a master.
